### DISCUSSION TITLE
how does third party works

### DISCUSSION THREAD

**Discussion by LibanTheCanadian** [BEEmod/BEE2-items#4193] [Tech Support / Q&A] [answered]
I just got some third-party items off the discord but do the player needs to get the items too?

**Comment by Areng14** [comment, marked as answer]
Nope they do not need to get the item

**Comment by Areng14** [comment]
Also by third party do you mean UCPs?

**Comment by LibanTheCanadian** [comment]
yea i think

**Comment by vrad-exe** [comment]
Please don't open issues to ask questions, this isn't a discussion forum

**Comment by LibanTheCanadian** [comment]
sorry
